django_app/utils/companies_house.py

```diff
@@ -47,9 +47,8 @@ def get_formatted_address(address_dict: dict[str, Any]) -> str:
         address_string += f",\n {town_or_city}"
     if postal_code := address_dict.get("postal_code"):
         address_string += f",\n {postal_code}"
+
     COUNTRY_DICT = dict(countries)
     if country := address_dict.get("country"):
-        # Do not add country to UK address - this is captured by location
-        if country not in ["England", "Northern Ireland", "Scotland", "Wales", "United Kingdom", "GB"]:
-            address_string += f",\n {COUNTRY_DICT[country]}"
+        address_string += f",\n {COUNTRY_DICT[country]}"
```

Comment:
I think this won't work with companies house address dict as the country returned is not "GB". Something like: 
```suggestion
        if country in ["England", "Northern Ireland", "Scotland", "Wales", "United Kingdom"]:
            address_string += f",\n United Kingdom"
        else:
            address_string += f",\n {COUNTRY_DICT[country]}"
```

Reply:
@morganmaerees good spot thanks, I've updated the code with the CH reference here - https://github.com/companieshouse/api-enumerations/blob/1cea0876a5bb6dc59d90694825c11cdfc0fc3634/constants.yml#L8